src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -519,6 +669,24 @@ def ne(self, y):
         """
         raise NotImplementedError
 
+    @staticmethod
+    def shares_sum(shares: List[torch.Tensor], ring_size: int) -> torch.Tensor:
+        """Returns sum of tensors based on ring_size.
+
+        Args:
+            shares (List[torch.Tensor]) : List of tensors.
+            ring_size (int): Ring size of share associated with the tensors.
+
+        Returns:
+            value (torch.Tensor): sum of the tensors.
+        """
+        if ring_size == 2:
+            return reduce(lambda x, y: x ^ y, shares)
```

Comment:
Hmmm, this will be without carry right so it will be inaccurate. Should we defer it to another PR? or raise a warning?

Reply:
The carry part is needed for local decomposition and composition for consistent shares, here we are computing a single binary tensor, we would not need it.